compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/AnchoredDraggable.kt

```diff
@@ -1652,6 +1651,8 @@ private class DefaultDraggableAnchors<T>(
     }
 }
 
+internal expect inline fun assertOnJvm(statement: Boolean, message: () -> String): Unit
```

Comment:
Intentional change of incorrect AOSP common fix. Please revert.

Reply:
Reverted. Will be upstreamed